### ISSUE TITLE
Custom Operators Zoo

### ISSUE THREAD

**Issue by wkcn** [wkcn/MobulaOP#87]
Hi there, this issue is to summarize some custom operators to be supported.
Please feel free to add it if you want any operator : )

- [x] RoIAlign
- [ ] Softmax
- [ ] Standard Convolution
- [ ] Bilinear/Trilinear Interpolation
- [ ] [Stand-Alone Self-Attention in Vision Models](https://arxiv.org/pdf/1906.05909.pdf)

**Comment by kohillyang** [comment]
I suggest several operators for reference:
1. CTC Loss, this OP is supported by mxnet, but before PyTorch 1.0, users must compile by themselves.

2. Deformable Convolution, Same as 1, users need compiling, and mxnet does not provide CPU implementation version.

3. Image wrapAffine based on Homography matrix. If this one is implemented, users can implement their image augmentations like image crop, translation, rotating and even scaling based on GPU.
